.nycrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@istanbuljs/nyc-config-typescript",
+  "cache": false,
+  "compact": false,
+  "reporter": ["text", "lcov"],
+  "excludeAfterRemap": false,
+  "all": true,
+  "temp-dir": "tmp"
```

Comment:
I was hoping I didn't need this location for the `.nyc_coverage.json` file, but I couldn't figure out how to keep it as deafult.